src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -55,6 +55,7 @@ type ExternalProps = {
     onTaskCreate: Function,
     onTaskDelete: (id: string) => any,
     onTaskUpdate: () => any,
+    onTaskView: (id: string, isCreator: boolean) => any,
```

Comment:
Do you have to propagate this even higher, to the ContentPreview or Preview Sidebar component props? Those are the top-level exports from BUIE.

Reply:
These props get passed in inside ActivitySidebarProps, which is inside ContentSidebarProps, which gets passed to ContentPreview